psalm-baseline.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="3.18.2@19aa905f7c3c7350569999a93c40ae91ae4e1626">
```

Comment:
What's the point of moving Psalm issues to a baseline file? We will have to update this line every time we upgrade Psalm, right? If yes, how?

Reply:
It's true that I think it's usually used when having a big baseline, when you don't want do deal with issues from the start, so I've moved the issue to `psalm.xml`. Out of curiosity, for updating the baseline: `vendor/bin/psalm --update-baseline`.